### ISSUE TITLE
Regex fails when field_name starts with a special character

### ISSUE THREAD

**Issue by eddoliveira** [shaffooo/django_nplusone#3]
If we have a model with some field like:
```
current_card = models.ForeignKey(    
    Card, on_delete=models.CASCADE, related_name="+", null=True
)
```
nplus one will fail with **"Python Regex Error : nothing to repeat at position.."**

The error occurs when field_name='+'.

```
def report_warning(self, model_name, field_name, relationship):
        """
        Use call stack to trace the statement responsible for N+1 and log it if applicable
        """
        for file, line, function, statement in extract_stack():
            if re.search(r'\b{field_name}\b'.format(field_name=field_name), statement):
                self.log_warning(model=model_name, field=field_name, relationship=relationship,
                                 file=file, function=function, line=line, statement=statement)
                break
```

**Comment by eddoliveira** [comment]
Added `re.escape(field_name)`
db739cf41d4faf9b49bf28c268a5bf5a157751fb

**Comment by shaffooo** [comment]
Thanks for the PR. The changes are released in `v0.0.6`
